server/src/main/java/io/crate/role/Role.java

```diff
@@ -117,6 +215,13 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (password != null) {
             password.toXContent(builder, params);
         }
+        if (parentRoles.isEmpty() == false) {
+            builder.startArray("parent_role_names");
```

Comment:
```suggestion
            builder.startArray("granted_roles");
```

It's role name and grantor, no?

Reply:
yes, thx, I forgot to change that after adding the grantor.